insights/parsers/yum_updateinfo.py

```diff
@@ -0,0 +1,31 @@
+"""
+UpdateInfo - command ``yum updateinfo list -C``
+===============================================
+Provides a list of available advisories
+"""
+from insights import CommandParser, parser
+from insights.parsers import parse_delimited_table, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.yum_updateinfo)
+class YumUpdateinfo(CommandParser):
+    """
+    Class for parsing the output of yum updateinfo list.
+
+    Output:
+        >>> [{ "advisory": "RHSA-2020:5206", "type":"Sec./Moderate", "package":"kernel-5.9.12-100.el8.x86_64"}]
```

Comment:
@semtexzv you are missing a sample input section and this section would be better as `Examples:` which you currently don't have.  Then you can add a test for the docstring examples.  Take a look at the [YumRepoList parser](https://github.com/RedHatInsights/insights-core/blob/863624a95054d7ee2eac2a4553b616c1ea569e18/insights/parsers/yum.py#L44) as an example of the above sections.  You can find the docstring tests for that parser [here](https://github.com/RedHatInsights/insights-core/blob/863624a95054d7ee2eac2a4553b616c1ea569e18/insights/parsers/tests/test_yum_repolist.py#L218).